e2e/app/page/featured-workspaces-page.ts

```
@@ -14,7 +16,10 @@ export default class FeaturedWorkspacesPage extends AuthenticatedPage {
 
   async isLoaded(): Promise<boolean> {
     try {
-      await this.waitForTextExists(PAGE.HEADER);
+      await Promise.all([
+        waitUntilPageContainsText(this._page, PAGE.HEADER),
+        waitUntilNoSpinner(this._page)
+      ]);
       return true;
     } catch (e) {
```

Why are you swallowing this exception message? Maybe log it at least.